kentik_api_library/kentik_api/requests_payload/labels_payload.py

```diff
@@ -82,12 +82,12 @@ def from_json(cls, json_string: str):
         labels = cls()
         for item in items:
             item["devices"] = _DeviceArray.from_list(item["devices"])
-            l = from_dict(GetResponse, item)
-            labels.append(l)
+            label = from_dict(GetResponse, item)
```

Comment:
Ditto (below too)

Reply:
Still to do

Reply:
Do You think label here is not readable?

Reply:
Responded: https://github.com/kentik/community_sdk_python/pull/112#discussion_r883578242

Reply:
The best solution: remove the local variable entirely. It is not needed:
```
labels.append(from_dict(GetResponse, item))
```

Reply:
Done